### ISSUE TITLE
Why was my IP blacklisted?

### ISSUE THREAD

**Issue by cypaul** [MicrosoftDocs/microsoft-365-docs#592]
This is all great but how do I determine why was my server blacklisted in the first place? It's happened to us twice. I went through the mail logs on my server and wasn't able to find anything suspicious. The server is not blacklisted anywhere else.

It would be great if you could provide more details on why this happened. Header of a malicious email would be a good start.

---
#### Podrobnosti dokumentu

⚠ *Neupravujte tuto část. Vyžaduje se k propojení problémů docs.microsoft.com ➟ GitHub.*

* ID: 390d461b-7fce-d4ac-1c90-97b7b81dc0c2
* Version Independent ID: 9f8c7285-627f-13c6-1c7a-88fa7e14ed86
* Content: [Use the delist portal to remove yourself from the Office 365 blocked senders list](https://docs.microsoft.com/cs-cz/microsoft-365/security/office-365-security/use-the-delist-portal-to-remove-yourself-from-the-office-365-blocked-senders-lis#feedback)
* Content Source: [microsoft-365/security/office-365-security/use-the-delist-portal-to-remove-yourself-from-the-office-365-blocked-senders-lis.md](https://github.com/MicrosoftDocs/microsoft-365-docs/blob/public/microsoft-365/security/office-365-security/use-the-delist-portal-to-remove-yourself-from-the-office-365-blocked-senders-lis.md)
* Service: **o365-seccomp**
* GitHub Login: @MSFTTracyP
* Microsoft Alias: **tracyp**

**Comment by danpadure** [comment]
the delising process does not work.  I did not get any email. Tried with different emails: yahoo, gmail
no email is being sent!

**Comment by mypil** [comment]
@cypaul @danpadure -  Thank you for submitting feedback.

I think the best way forward is if you can open a **[service ticket](https://docs.microsoft.com/office365/admin/contact-support-for-business-products)** in your tenant so this can get resolved ASAP. Based on outcome, let me know if it is something that can be called out in the docs.

If you don't mind please keep us posted here on the resolution and I really want to get this information you are discovering back into the docs.

**Comment by cypaul** [comment]
@mypil - Thanks for your reply. I am not a customer of Microsoft 365.

I have a mailserver which occasionally sends emails to Office 365 users. Twice in the past month my server's IP address got blacklisted and users got bounces like this one:

    host eon-cz.mail.protection.outlook.com [104.47.0.36]
    SMTP error from remote mail server after RCPT TO:<info@eon.cz>:
    550 5.7.606 Access denied, banned sending IP [12.13.14.15]. To request removal from this list please visit https://sender.office.com/ and follow the directions. For more information please go to  http://go.microsoft.com/fwlink/?LinkID=526655 (AS16012609) [HE1EUR01FT057.eop-EUR01.prod.protection.outlook.com]

So I went ahead and delisted (the process worked for me). The real question is though - why was my server blacklisted? How do I prevent it from being listed again?

There are not many emails from my server going to Office 365. I checked them all and none of them looked unsolicited. What else can I do?

**Comment by mypil** [comment]
@cypaul - Thank you for this additional information. Let me loop in the doc owner @MSFTTracyP to get his insights.

**Comment by mypil** [comment]
@MSFTTracyP - Can you please share your insights on this issue?

Thank you.

cc: @kenwith

**Comment by danpadure** [comment]
Just so you know, I managed to delist my IP. 
After trying with so many emails (business, gmail, yahoo), I finally asked a friend with an office365 account to try and he immediately received the confirmation email

**Comment by BarrLover2019** [comment]
To be honest I didn’t mean to do that and I have no idea what it is I did I
don’t understand any of this stuff on there! I’m sry if that was a negative
thing I didn’t know!

**Comment by picklesrein** [comment]
The captcha does not work. Tried this for 30 minutes in different browsers. Asked three colleagues to try it. Apparently our company is made of cyborgs.

**Comment by bobhwasatch** [comment]
Your captcha is hilariously impossible to solve. I just don't care that much. Screw you.

**Comment by AdminSoftTeco** [comment]
Step 2: An email with instructions has been sent to *my email*. Check your Inbox and your junk folder for the email. If you don't see the email within one hour, start over at Step 1 but use a different e-mail account.

but nothing came to my email ( I tried more than 5 times)

**Comment by bcebulla** [comment]
I neither get a verification email from this delist portal. I have tried this two dozen times from 5-10 different email accounts, nothing works. Support tickets do not seem to work either, customer support just keeps pointing us to this page--it's an unsolvable loop!

**Comment by danpadure** [comment]
ask a friend with a office365 account to delist you.  his email will work

**Comment by bcebulla** [comment]
Thanks for suggestion Dan. Still doesn't work. Tried an office365 account and a office365 admin account. So frustrating.

**Comment by bcebulla** [comment]
Solved this Dan--it is unrelated to having an Office365 account. 

The problem is that MSFT's form is sending a misconfigured email (https://twitter.com/snootgirl/status/1177037875260612608), so Gmail is blocking MSFT's email, which is super ironic. 

I solved this by using a made up email address at http://www.yopmail.com/en/, they'll accept anything. Sure enough, got the email immediately.

**Comment by Jaimer** [comment]
Thanks, used yopmail and received the mail immediately.

**Comment by DNied** [comment]
Your email with delisting instructions did not make it through my SMTP server. Here's what happened:
```
Nov 25 11:47:07 smtpd[12057]: 2c9323b359d01a8a smtp event=connected address=40.107.79.74 host=mail-eopbgr790074.outbound.protection.outlook.com
Nov 25 11:47:08 smtpd[12057]: 2c9323b359d01a8a smtp event=starttls ciphers="version=TLSv1.2, cipher=ECDHE-RSA-AES256-GCM-SHA384, bits=256"
Nov 25 11:47:09 aretusa smtpd[12057]: 2c9323b359d01a8a smtp event=bad-input result="500 5.0.0: Line too long"
Nov 25 11:47:09 smtpd[12057]: 2c9323b359d01a8a smtp event=closed reason=quit
```
You blacklisted my personal, secure, well-behaved SMTP server and you can't even send an email? What a farce.

**Comment by urbaman** [comment]
Hi,

Having the same problem:
1) IP blocked
2) My business address does not receive the confirmation email (can't debug why my smtp does not accept them)
3) My gmail address receives it, I finish the delisting operation, but no delisting happens (stil getting mails bounced with IP banned

How can I get trhou this?

Thank you.

**Comment by MSFTTracyP** [comment]
Following up on this issue internally. Thank you all for reporting it. If you know of others having delist portal issues *please direct them to this thread to report*. Likewise, if there have been any developments in your cases, please let me know in this thread.

**Comment by urbaman** [comment]
Hi,

My issue was with greylisting (I hard both Plesk's greylist and postgrey active on my mail server).

As soon as I deactivated them, everything got working smoothly, I delisted the IP and my mails got delivered to office365 accounts.

**Comment by DNied** [comment]
What Microsoft is doing is very clear: they're blacklisting everyone by default, and they're shifting the work on the admins to have their server delisted. We're supposed to do some unpaid work for Microsoft, so that they get a selected pool of approved SMTP servers for free. They're the usual **scumbags** they've always been.

But here's what happens instead: I'll contact your customers via my ISP's SMTP server, and let them know that I'm having difficulty reaching them because their pricey Outlook subscription is cutting them out of the regular email circuit. Let's see how they like it. Go fuck yourself, Microsoft.

**Comment by fredt21** [comment]
I'm also unable to delist.  I received the email, and successfully requested delisting twice.  Each time I received a message that the delisting was successful and that it may take 30 minutes to update.  I waited over 30 minutes both times and I am still getting rejected.  Is there a way to tell if I am getting re-listed, perhaps due to a configuration issue?  I've resolved a couple of config issues in the last few days, one with SPF and one with the reported server name.

**Comment by Megashape** [comment]
Same issue here, can't delist - the tool from Microsoft is useless.

**Comment by mypil** [comment]
Related issues #674 & #712

**Comment by ex-bart** [comment]
There multiple issues in this thread, I think I can say something about the mails from the delisting portal not getting delivered.

When I tried delisting, I got this in my exim's rejectlog:
```
2019-12-13 10:38:01 1ifhOb-0001j0-Vk H=mail-dm6nam12on2089.outbound.protection.outlook.com (NAM12-DM6-obe.outbound.protection.outlook.com) [40.107.243.89] F=<no-reply@microsoft.com> rejected after DATA: maximum allowed line length is 998 octets, got 3172
```
The 998 octet limit corresponds to the 1000 octet limit from [RFC 5321 §4.5.3.1.6 "Text Line"](https://tools.ietf.org/html/rfc5321#section-4.5.3.1.6), I guess, one counting the final `<CRLF>`, the other one not counting it.  After removing the limit and retriggering the delisting, I received a mail with the following body:
```
----boundary_0_71906a35-e6a6-4c10-a9bb-7c769e2c7e42
Content-Type: multipart/related; type="text/html";
 boundary=--boundary_1_2e3f11f4-1877-4e8f-aad1-6998e14dc347

----boundary_1_2e3f11f4-1877-4e8f-aad1-6998e14dc347
Content-Type: text/html; charset=utf-8
Content-Transfer-Encoding: base64

[CENSORED 852 bytes of base64-encoded HTML on a single line]
----boundary_1_2e3f11f4-1877-4e8f-aad1-6998e14dc347
Content-Type: image/gif
Content-Transfer-Encoding: base64
Content-ID: <HeaderImage>

R0lGODlhRAIZALMAAKCgoPHx8WlpacnJyZKSklxcXLu7u0FBQU5OTuTk5K2trdbW1oWFhXd3dzMzM////yH/C1hNUCBEYXRhWE1QPD94cGFja2V0IGJlZ2luPSLvu78iIGlkPSJXNU0wTXBDZWhpSHpyZVN6TlRjemtjOWQiPz4gPHg6eG1wbWV0YSB4bWxuczp4PSJhZG9iZTpuczptZXRhLyIgeDp4bXB0az0iQWRvYmUgWE1QIENvcmUgNS41LWMwMjEgNzkuMTU0OTExLCAyMDEzLzEwLzI5LTExOjQ3OjE2ICAgICAgICAiPiA8cmRmOlJERiB4bWxuczpyZGY9Imh0dHA6Ly93d3cudzMub3JnLzE5OTkvMDIvMjItcmRmLXN5bnRheC1ucyMiPiA8cmRmOkRlc2NyaXB0aW9uIHJkZjphYm91dD0iIiB4bWxuczp4bXBNTT0iaHR0cDovL25zLmFkb2JlLmNvbS94YXAvMS4wL21tLyIgeG1sbnM6c3RSZWY9Imh0dHA6Ly9ucy5hZG9iZS5jb20veGFwLzEuMC9zVHlwZS9SZXNvdXJjZVJlZiMiIHhtbG5zOnhtcD0iaHR0cDovL25zLmFkb2JlLmNvbS94YXAvMS4wLyIgeG1wTU06T3JpZ2luYWxEb2N1bWVudElEPSJ4bXAuZGlkOjIwNWJlYmJmLTliMWItY2U0OC1iODJlLTI1N2JkYjFhM2JkOCIgeG1wTU06RG9jdW1lbnRJRD0ieG1wLmRpZDo2OUNBNEEzQTgyNUExMUU0QjBBNjkwOUQ0REY1RTRBRCIgeG1wTU06SW5zdGFuY2VJRD0ieG1wLmlpZDo2OUNBNEEzOTgyNUExMUU0QjBBNjkwOUQ0REY1RTRBRCIgeG1wOkNyZWF0b3JUb29sPSJBZG9iZSBQaG90b3Nob3AgQ0MgKFdpbmRvd3MpIj4gPHhtcE1NOkRlcml2ZWRGcm9tIHN0UmVmOmluc3RhbmNlSUQ9InhtcC5paWQ6MjA1YmViYmYtOWIxYi1jZTQ4LWI4MmUtMjU3YmRiMWEzYmQ4IiBzdFJlZjpkb2N1bWVudElEPSJ4bXAuZGlkOjIwNWJlYmJmLTliMWItY2U0OC1iODJlLTI1N2JkYjFhM2JkOCIvPiA8L3JkZjpEZXNjcmlwdGlvbj4gPC9yZGY6UkRGPiA8L3g6eG1wbWV0YT4gPD94cGFja2V0IGVuZD0iciI/PgH//v38+/r5+Pf29fTz8vHw7+7t7Ovq6ejn5uXk4+Lh4N/e3dzb2tnY19bV1NPS0dDPzs3My8rJyMfGxcTDwsHAv769vLu6ubi3trW0s7KxsK+urayrqqmop6alpKOioaCfnp2cm5qZmJeWlZSTkpGQj46NjIuKiYiHhoWEg4KBgH9+fXx7enl4d3Z1dHNycXBvbm1sa2ppaGdmZWRjYmFgX15dXFtaWVhXVlVUU1JRUE9OTUxLSklIR0ZFRENCQUA/Pj08Ozo5ODc2NTQzMjEwLy4tLCsqKSgnJiUkIyIhIB8eHRwbGhkYFxYVFBMSERAPDg0MCwoJCAcGBQQDAgEAACH5BAAAAAAALAAAAABEAhkAAAT/8MlJpWkNpMq7/2AojmRpnmiqrmzrvnAsz3Rt3+tCHE7fIwQDbkgsGo/IpHLJbDqdAUDBR60qntisdsvter/g56VKpgLC6LR6zW673yAdr4rILMpnl2HKkAAQDgpRC3CFhoeIiYpFAHQMChsTeCADDHMODUIVBg4FAGeNAgAGjQ0uAZY9B3kUCQw+CIQPCgK1tleLubq7vL0ijT6aHJMewBkAqX0UBQcBEwcIEwsNAy4DQJ8CmBQLBwcEnwzVD6G2Ari+6err7GnAPePDZKwVjQWyEgENDgQUDgL+AM4I4GyCNnwIDuCj0Kidw4cQIyp55yBeBWIVEjg4EKnClHj//wISaTROQSAP+ySqXMmy5QmKFv3N60DAAT0KnPooaITg0ycHPTUkAGBRQYNaBAqSq9UA3YdGsgpE8yBgqsurWLNGhOkB4zIHSodF01bGx4ABNiUkmFKgVsUHAdhW5QciAIIC+Rz0CXC2Qi2tgAMvItiB8IMEYXEsEFAgJhKuHbxKstpBmySBEkJKQJtnCroAG/ah2xczwQAAOwQU5PzuADqynQh0FEy79peaaSngNpDAW5HenarNPgJZXpWbmTFXdnBZ5Oa0JpFrVKaWLkMfBdCZlKpggAIemhh8IgBIoe3z6LHgVv4AkAMDAf4SqdkZ7EQqjjPPHMbeL/Pkzj3A2f8DrwxHjk1nJajZBKad9UoBzjTSjDSddBBKehhmeERNPOBjz3twhWUYXPlMMKKJiZ1YE3wB9JAYI/h1tV8FPIDgG4DNPXeGZRxQREV/14FinUH/cTCFgRomqWQLNe1DnQAHaANfhQ8spsoANTEwRQBWbtRPlWQJtMcPZ0xRBl6PxRjZjBTsI0wFd5iC45wD8lgPggmetVBkAA1IgZ135rfkoISOsGJCEtyh5Xst4gUcAwQIYABuBdzDA6R8PKANpP1w8s0OgQCgTQME4AZOEsVdxOYEJgH5AGk50pnWK3vOctIJIbVInQSAUlBgocAGG8KKNV1B6xRT4rUPPTUJtGz/Pt606EBHUj6H14pwuXifWTIeR5VePS74gLgLDsiJnK1QWQInX0KplEbotgKNsPTWO8GKdwAU5QPINtqeffdad+QEH71yAAMEaSsBDwFgK+2LI/nAAJLjrmriFOecRUsnYZGLmZ+jEjJUNc0SEoAClL0KgMgARGttNQNMQUglQpwMyJv25jwottoUyy+jHCssQU1fDizBR+QAAqHQDDssNHGeQCyJxSaC6sMqiXk8gZ8NU2HeA7j5wN4rP872HSzj3EEFAjjr7LaG2Ho6Vb8VasOsdc/CdSOv7yGbaIVOA4xEAAbsNADFknHAlwLdeTDAQo+beHgrpAwQVgKVI7nAi2mTK855RqcpUOvbpKeH7b9FA42Xp5BKSvRzBxMwBQB2lZrQyV6SB+LpfCBnhAGpICAAOH0lXvrxyLNxOlRHqy4Byj8k8LoFZvbUPFC4QI/90CDOwoOrRpwMW1k9+J78+eh3ISIFBaWo1Isnhqh4ihVIrcRQ7jkCX/r89+9/DHLAxMr+R8ACGpACEQAAOw==
----boundary_1_2e3f11f4-1877-4e8f-aad1-6998e14dc347--

----boundary_0_71906a35-e6a6-4c10-a9bb-7c769e2c7e42--
```
(Censoring of the base64-encoded HTML content by me)

So this happens because of an unnecessary (i.e. eye-candy only) image that is included in the mail, and the base64 encoded content not being folded into shorter lines as would be customary.

The in my case limit came from the default configuration of Debian's exim.  Not quite sure why they put the RFC-required minimum into their default configuration, but I guess I'm gonna report this there as well.

**Comment by ex-bart** [comment]
In my previous comment I assumed that Debian based the decision to reject messages with overlong lines on [RFC 5321 "Simple Mail Transfer Protocol"](https://tools.ietf.org/html/rfc5321), which just recommends limiting line length to 1000 octets, including `<CRLF>`.  In fact, Debian based its decision on [RFC 5322 "Internet Message Format" §2.1.1. "Line Length Limits"](https://tools.ietf.org/html/rfc5322#section-2.1.1), which defines much stricter limits of 998 (required) and 78 (recommended, both excluding `<CRLF>`), see [Debian Bug#797919](https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=797919).  The decision to not even accept such messages was because trying to pass them on to the next SMPT server can lead to severe problems with other mail destined to the same next SMTP server (for instance if that server is gmail).

**Comment by VocalizedBus** [comment]
Facing same problem right now. Setting up a squeaky-clean new VPS with 10 WordPress installations that send system-generated emails only to myself (I am an Office 365 subscriber). It sends no other emails to no other people (just login & WordFence warning emails, and only to me). I did not receive a single email, yet my server IP is already blacklisted. 

I have submitted through the MSFT form to have it removed but after a few days I'm still on the list. How long should we expect to wait to be removed, and how many times should I have to submit before I can realistically expect to be removed?

Thank you.

**Comment by VocalizedBus** [comment]
Update: Apparently we've already been removed, however our emails are still being rejected. When I fill out the Office365 Anti-Spam IP Delist Portal form, I get the following response: 

[quote]The IP address in question is not currently blocked in our system. Please refer to the email message you received from Microsoft and follow the steps it suggests.

If you're trying to delist an IP address blocked in Hotmail.com/Live.com/Outlook.com, please use the following link.
https://support.microsoft.com/en-us/getsupport?oaspworkflow=start_1.0.0.0&wfname=capsub&productkey=edfsmsbl3&locale=en-us&ccsid=635900597399910848

If you're trying to delist a banned or blocked sender, please open a support ticket from your tenant.[/quote]

However the issue is we are never receiving the emails in the first place. I'm only seeing these error messages on my server's outbound exim logs. It's as if the emails are being blocked from out Office 365 tenant before they even hit the server; the emails aren't even landing in quarantine.

Thank you.

**Comment by eepotter** [comment]
I'm interested in the original poster's (as yet unanswered) question - what causes an IP address to end up on this list?  I set up a server that worked for a year before being blacklisted here; I did the request and it worked fine for a month and then got blacklisted again. I don't send spam, and Microsoft is the only place blocking the server. It would be nice to know how to prevent this from happening again, aside from advising clients to not use Office 365.

**Comment by kmorwath** [comment]
I'm interested too. I just migrated my mail server from a 2008R2 machine to a 2019 one, on a different IP (checked with Spamhaus and Barracuda, and it is "clean"). All DNS settings and SPF/DKIM/DMARC records were updated and are correct.
Tested with outlook.com and mails were received. Still Office365 blocked mails - and not all users are able to understand what the returned error means.
Does Office365 blocks IP that never sent mails and have no "reputation" immediately? Why? Why not verify SPF/DKIM and assign an initial reputation, putting the new IPs in a list of sources to be monitored closely for bad behaviours?
Putting the onus to remove the block on non-technical users is not nice - and it can block important mails.
I've registered with SNDS, but that's another complication - what if all largest email provider ask the same? It may even been seen as an anti-competitive behaviour to block smaller players and people using their own servers and move people to the largest cloud systems. Maybe it's time to file a complain to antitrust authorities.

**Comment by EmptyNull** [comment]
I would like to know why our mail server keeps getting listed again and again.

**Comment by haberzeth** [comment]
Same Problem here. Delist Portal didn't work. 
Mailed the problem to delist@messaging.microsoft.com and wait now a week to get delisted.

**Comment by arenstar** [comment]
How can Microsoft start banning IP's without giving any reasoning, then build a tool to get delisted that does not conform to standard SMTP rules ( which somehow only works for office 365 clients )?

**Comment by no-test** [comment]
Why was the IP blocked? 

Is microsoft incapable of answering the question? sending an email saying an ip was blocked WITHOUT giving a reason: seriously?

ANY other blacklisting service would NEVER block ip without giving a reason, there you go that microsoft for you.

Half a year and the posters question is STILL unanswered.

**Comment by kristjanpraakle** [comment]
Same Problem here. And also Delist Portal didn't work.
Mailed the problem to delist@messaging.microsoft.com  waiting for feedback

We also do not have the answers: Why was the IP blocked? question is STILL unanswered. Reason is unknown. Also the Delist Portal did not offer any info.

We have good reputation for almost 10 years not once being in any blacklists.

**Comment by sozzled999** [comment]
This is bizarre but at least I feel not alone.

I have been through exactly the same sequence.
We have moved our email server to a new ip as part of a location move, now we get the bounce message.
Step 2 to receive an email to unblock did not work.
Using an alternate email not on our mail server / therefore our IP - did.
I have received a message saying we are no longer banned, but emails are still being blocked 3 hours later.  I tried again to delist but was told we are no longer listed.

So why are you still blocking our emails Microsoft?

This is not a good look.  I am non techie, have worked all this out via google and chat forums.
Glad I seem to be on the right path but nervous that it does not yet seem to be fixed.

If someone could actually represent Microsoft here and explain how to fix this that would be useful!!!

**Comment by dsaul** [comment]
I go to the site, it tells me my IP is not banned, even though I get a bounce back when emailing office 365. I submit the form, microsoft emails me telling me to use the site that doesn't work. I reply to them telling them it doesn't work, I get a bouceback stating I am banned. This is a Kafka nightmare.

**Comment by alehaa** [comment]
For my IP, the [delist portal](https://sender.office.com) tells me, that it's not listed, too. However, it also states for delisting the IP for Outlook.com, Hotmail and Live, you need to use [another form](https://support.microsoft.com/en-us/supportrequestform/8ad563e3-288e-2a61-8122-3ba03d6b8d75). I tried this form and got the response, my IP does "not qualify for mitigation". The reply asks me to check my mailserver's compliance with [their guidelines](http://mail.live.com/mail/policies.aspx), which I'm sure it does (even GMail does accept my mail).

The funny thing is, they even offer helping me with this issue, by simply writing them a reply on the ticket mail. However, I can't, as their ticket system also uses the blocklist my server's IP is on. Unfortunately the support is not available via chat.

**Comment by joloco** [comment]
I run a server which has very little mail activity (a dozen emails a day would be busy) yet Microsoft's algorithm has somehow decided that there is suspicious activity coming from my IP address. 

Entering my server's IP into block-list checking sites shows nobody else is flagging it, and I've even searched through the logs, which show that there are hardly any emails to Microsoft domains, and those few emails are definitely manually sent by one of my few users.

So I fill in the forms as suggested, eventually getting a pre-written reply – "not qualified for mitigation", "recent activity flagged by our system as suspicious", etc.

Yet repeated requests to see *any* evidence or information about this supposed suspicious activity are met with still more pre-written emails suggesting I sign up to JMRP and SNDS (have been for years, no activity showing, no reports), implement DKIM and SPF (already implemented). 

As others here have said – if there is evidence of abuse originating from our servers, please share it with us so we can fix it! If there's none, you should unblock our IP.

**Comment by alehaa** [comment]
Apparently my IP got delistet without any notice, even if I was told my server doesn't qualify ...  I really think this process needs a little bit more transparency.

**Comment by joloco** [comment]
My IP was removed from the block list, after a bit of back-and-forth with actual humans (as opposed to auto-reply bots) at Microsoft. Took about 5 days in total. 

I told that I wouldn't be informed why it was put on the block list in the first place (something to do with privacy reasons). None of Microsoft's own tools ever gave me any information about why it was added in the first place.

So yes, as @alehaa says, this process does need more transparency!

**Comment by rbplatt** [comment]
I use SendGrid.  Used their shared IP in the past, but just upgraded to private IP.  Put all the security measures in place including Reverse DNS.  We only use this for two-factor authentication.  As soon we enabled the private IP, Office 365 put's it on the block list (we're Office 365 subscribers).  I agree with "DNied" above--we seem to be doing Microsoft's work for them.  Still trying to get de-listed.

**Comment by geeksonline** [comment]
I can confirm that Microsoft do in fact block EVERYONE using smtp relay - even if all connector / spf records are same. What I can also say is their delisting doesn't work, you almost always have to raise a ticket and even if you manage to get delisted, your address will get banned again. This is absolutely ridiculous from Microsoft. Business systems stop running (like helpdesk ticket systems). Surely someone with half a brain inside the company can realise the affect this has on their customers.

**Comment by MSFTTracyP** [comment]
Looking into issues with delist portal. Thank you to everyone contributing to this thread.

**Comment by joloco** [comment]
Thank you Tracy! Anything which could provide us with a clue as to why our IP was blocked would be useful – even just a date and time of when these apparent spam emails were sent, so we can check our logs. Because at the moment we have no idea what is said to have happened, or when!

**Comment by soriyath** [comment]
I can confirm that Office 365 added my IP address to their Deny list for no reasons at all (it works perfectly with other mail providers).
I can also confirm that on August 18 they still send bad emails that don't pass basic spam filters.

excerpt from my logs:
```bash

Aug 18 10:57:22 xxxxx postfix/smtpd[XXXXX]: connect from mail-mw2nam10on2080.outbound.protection.outlook.com[xx.xxx.xx.xx]
Aug 18 10:57:22 xxxxx postfix/smtpd[XXXXX]: Anonymous TLS connection established from mail-mw2nam10on2080.outbound.protection.outlook.com[xx.xxx.xx.xx]: TLSv1.2 with cipher ECDHE-RSA-AES256-GCM-SHA384 (256/256 bits)
Aug 18 10:57:23 xxxxx postfix/smtpd[XXXXX]: XXXXXXXXXX: client=mail-mw2nam10on2080.outbound.protection.outlook.com[xx.xxx.xx.xx]
Aug 18 10:57:23 xxxxx postfix/cleanup[XXXXX]: XXXXXXXXXX: message-id=<f7e31b31-280b-4aed-a765-0c3ba63e11a9@BN1BFFO11FD003.protection.gbl>
Aug 18 10:57:23 xxxxx opendkim[XXX]: XXXXXXXXXX: mail-mw2nam10on2080.outbound.protection.outlook.com [xx.xxx.xx.xx] not internal
Aug 18 10:57:23 xxxxx opendkim[XXX]: XXXXXXXXXX: not authenticated
Aug 18 10:57:23 xxxxx opendkim[XXX]: XXXXXXXXXX: failed to parse Authentication-Results: header field
Aug 18 10:57:23 xxxxx opendkim[XXX]: XXXXXXXXXX: no signature data
Aug 18 10:57:23 xxxxx opendmarc[XXX]: XXXXXXXXXX: microsoft.com fail
Aug 18 10:57:23 xxxxx postfix/cleanup[XXXXX]: XXXXXXXXXX: milter-reject: END-OF-MESSAGE from mail-mw2nam10on2080.outbound.protection.outlook.com[xx.xxx.xx.xx]: 5.7.1 rejected by DMARC policy for microsoft.com; from=<no-reply@microsoft.c
om> to=<XXX@XXX.XX> proto=ESMTP helo=<NAM10-MW2-obe.outbound.protection.outlook.com>
Aug 18 10:57:23 xxxxx postfix/smtpd[XXXXX]: disconnect from mail-mw2nam10on2080.outbound.protection.outlook.com[xx.xxx.xx.xx] ehlo=2 starttls=1 mail=1 rcpt=1 bdat=0/1 quit=1 commands=6/7
```

You see the line stating "microsoft.com fail": the Delist tool is sending email that can't pass the spam filters...

**Comment by Varstahl** [comment]
I've lost count of how many times I've been directed to https://sender.office.com/, last of which today, just to get blocked by a "temporary" error.

![image](https://user-images.githubusercontent.com/284077/110646360-1e2a0280-81b7-11eb-9423-a2f82429a62e.png)

Either I'm the unluckiest admin on the planet, or that service crumbles more than a cookie. The no-auth mails are just the cherry on top. Also, tried different mail providers on top of my own, always fails at that point, doesn't even work with a 365 mail account.

**Comment by PaulMoore-PaymentsFoundry** [comment]
Same error here for last 2 days.  Immensely frustrating and support were absolutely useless.

**Comment by arhoads76** [comment]
I'm having the same issue as well.  Tried the https://sender.office.com/ portal too many times to count.  Tried an online chat session with Microsoft Support, but they kept insisting that they needed to start a screen share and look at my Outlook settings.  Is there a better way to contact support to get a human?

**Comment by floeff** [comment]
Same here.

SNDS shows all IP are clean, http://sender.office.com shows the same, all other blacklists don't yield any results for that particular IP either. DKIM and SPF are configured properly, so are A and PTR records.

The error message is: "550 5.7.511 Access denied, banned sender[xxx]. To request removal from this list please forward this message to delist@messaging.microsoft.com. For more information please go to http://go.microsoft.com/fwlink/?LinkId=526653."

I just got that IP assigned recently, so likely the previous owner created the problem. I sent in my full contact data and offered proof that the IP assignment is rather recent. The last interaction with the antispam team was on May 2, with feedback promised within 24 hours, but no response so far. There's currently a ticket open with the escalation services, but no news since Thursday either.

**Comment by geeksonline** [comment]
Keep chasing them up, that's all you can do. Unless your sending emails through Microsoft Outlook, you will almost definitely be blacklisted for relaying - even with everything set up 100% correctly (like it sounds you have done).

**Comment by Varstahl** [comment]
> The last interaction with the antispam team was on May 2, with feedback promised within 24 hours, but no response so far. There's currently a ticket open with the escalation services, but no news since Thursday either.

That's absolutely normal. In my case they replied with maximum times of up to 2 weeks, but I think it went past that. But, truth be told, they did follow it up and resolve it, so just give them time to address it. It's weird times for everyone, after all.

**Comment by MSFTTracyP** [comment]
I tracked down some pros inside of the company for the final word on this complex thread (thank you for the time you put in and your comments on this topic). I asked them to collect together some of the main questions in this thread and answer them. Here's what they had to say (and another strong reason to go to support if / when Strange Things Happen).

Can you tell why my IP is on your block list?
====================================
We get this question a lot! It's our responsibility to protect our customers and partners, and exposing such detail increases attack risk and abuse by bad actors. It's a good idea to check the server for signals of compromise, review other online blacklists, and if the server is clean, use the delist portal to remove it from the blocked senders list.

Does Microsoft block any new IP automatically?
==============================================
No. Absolutely not! Sometimes, IP addresses or ranges change hands, and past abuse may impact new owners. Scan the server for signals of compromise, and if it's clean, use the delist portal to remove it from the blocked senders list.

I didn't receive the validation email or unable to delist my server.
===================================================
Wait up to 24 hours while we remove the IP from block lists across our entire infrastructure. Contact support with the error message and the IP address and we'll be happy to take a look if the IP is on our block list after 24 hours. If you need Support to help with your situation, don't hesitate to contact us!

Again, this has been an enlightening and important topic to surface to our teams. Thank you to everyone for their input. Reach out to Support where needed. They are ready and willing to assist with issues like this.

Thank you!
MSFTTracyP

**Comment by joloco** [comment]
Thanks for tracking down this issue deeper within MS, Tracy – but I'm sorry to say that this doesn't answer many of our questions at all!

My situation – and that of many others – is a long-existing server with no evidence of spam (and in my case, very light email use overall) suddenly gets blocked by MS for abuse, and users (or, on my server, my single user) can no longer send emails to Hotmail or Outlook accounts.

There are no signs of misuse on the server, the IP isn't listed on any of the email abuse lists. No details are given by MS, no guidance is offered as to what might have happened. Only after going back and forth in a long and tedious dance with support robots and humans at MS is the server eventually unblocked.

So your answer doesn't actually answer anything at all, it just reiterates the standard information we're offered elsewhere. I appreciate that you're just passing on the information you were given, but there's absolutely nothing new there that isn't already on the various help pages already.

**Comment by MSFTTracyP** [comment]
@Jolco, the only other advice I can give you is to go through Support for these situations. It's a technical issue that  honestly needs a support ticket, and, optionally, a Tech Community thread, rather than to raise this as feedback change needed to the documentation. Unfortunately, the writing team cannot address your issue.

**Comment by chrcoluk** [comment]
I will be blunt, I think simply put the Microsoft filtering system is broken.  It is as simple as that.

There is a number of flaws I have identified.

1 - They wont provide technical information as to how the filtering got triggered, they probably do this as they believe they dont want to help spammers get round their systems, but the problem with this approach is that legitimate emails wont get delivered which is a far worse problem than letting spam through.  I would rather receive 100 spam vs missing 1 single proper email.
2 - If I enable even low filtering in Outlook desktop application, it has a ton of false positives, and this is on their weakest filtering, I dont know what the differences are between Outlook filters and their own remote server filters, but it does show how bad the filtering is.
3 - My IP has been listed so many times I can no longer even delist it, when they did delist in the past they used to refuse to make it permanent so of course the false positive rule they have that listed it, kicked in again.

Effectively I expect they whitelist gmail etc. to avoid mass complaints, and it leaves those of us with personal run SMTP servers getting caught again and again by the mess of a setup they have, they wont care as we are all too small players, they will just make sure the large email providers work, which is likely via whitelisting.

I had to get a new ip range to get round the problem and so far its not triggered again, but I may have to consider using gmail or something to send my emails to microsoft hosted accounts.

Note like many others my ip's are not blacklisted on any public RBL's and I dont have delivery issues to gmail etc.

**Comment by DBMandrake** [comment]
I wanted to vent my frustrations and add to the choir of discontent at Microsoft's abysmal IP address blocking tactics for mail being sent to Office 365 hosted/filtered email domains.

I work for a primary/secondary school, a large percentage of the people we need to contact either use Office 365 for email directly, or use the Microsoft Protection front end filtering for their own servers.

As of yesterday our mail server IP is blocked for the third time in 4 months and it is getting extremely tiring and frustrating. As with the previous two times we are NOT blocked by any public block lists or anyone else other than Office 365.

Delisting via the sender.microsoft.com form DOES NOT WORK as a means of being delisted. I can go through the process right to the end then have the request denied, with a button to "escalate" to Microsoft support but this does NOTHING. The IP address will never be delisted and nobody will ever contact you back. I can say this with confidence after two previous episodes. Complete waste of time using this form.

After the first incident I registered with the "Smart Network Data Service" as I'm sure some of you have done, this does NOTHING to help with this issue as all three times we have been blocked including right now NOTHING is listed as being wrong on the SNDS. We're all green there. I am also subscribed to the associated "Junk Mail Reporting Service", this has reported NOTHING so is also worthless.

To be honest I wonder if SNDS and JMRS are nothing to do with Office 365 and only apply to Outlook.com - we are not blocked for recipients at Outlook.com / live.com etc, only Office 365, so they must be two separate systems. So forget about SNDS and JMRS - they won't help you either.

On the previous two occasions I have tried two different routes to get delisted. The first time I tried raising a support ticket through our own Office 365 tenancy - while it took about 4 days the problem was eventually resolved. Of course anyone who doesn't have an Office 365 tenancy couldn't do this, and technically it's not their responsibility because I'm the SENDER who is blocked, not the RECIPIENT at the tenancy.

The second time I tried to go the same route and the support people I got flat out refused to help me, saying that it was a completely different department that handled the Office 365 incoming email filtering and that they could do NOTHING to help me, despite a documented previous support ticket still being on the system showing a resolution of the problem!

The second time I followed the route of emailing a hand written delist request to delist@messaging.microsoft.com - this address is documented here:

https://learn.microsoft.com/en-us/microsoft-365/security/office-365-security/use-the-delist-portal-to-remove-yourself-from-the-office-365-blocked-senders-lis?view=o365-worldwide

This also took at least 4 days before I got a reply but I finally did get a reply and I was delisted. I asked for an explanation of WHY we were listed so that I could make sure we weren't doing anything that might cause us to be re-listed, but no explanation was forthcoming.

Yesterday I have filled in both the sender.microsoft.com form and clicked the escalation button AND emailed delist@messaging.microsoft.com - at this point all I can do is wait, and try to explain to staff and senior management why the email they urgently need to send to parents and business associates of the school can't go through.

The reality is so many companies use Office 365 for email now (including several we need to contact on a daily or weekly basis) that Office 365 deciding to block you for no good reason has a major effect - I would estimate over 40% of the companies we deal with use Office 365 as a front end or as their mail system.

At no point will Microsoft support ever divulge WHY a server's IP address has been blocked - just giving platitudes like "make sure your server is not sending spam", err ok, already checked that.... If it was, we'd be on multiple other blocklists anyway.

Totally unacceptable Microsoft - for anyone trying to get this sort of problem resolved Microsoft is a big faceless corporation who simply DOES NOT CARE that they are causing material harm to organisations and businesses that RELY on email to function properly.

Even though we are a Microsoft CUSTOMER both for Windows Server and Desktop licenses, and an Office 365 customer ourselves (although we don't use our Office 365 tenancy for email) we still get the same useless treatment as everyone else trying to be de-listed by Office 365.

End of Rant.
